### ISSUE TITLE
Ownership of Information

### ISSUE THREAD

**Issue by ianjkalin** [CommerceGov/Policies-and-Guidance#12]
Regarding the section:

“DOC has relinquished control of the information and no longer has responsibility to protect the integrity, confidentiality, or availability of the information.” 

Concern expressed from open source community:

You can release something on an “as is” basis. That’s fine. That’s a staple of any open source license (disclaiming warranties of merchantability of fitness for a particular purpose).  But this particular sentence simply isn’t true, and especially isn’t true if I download the code via a quarterly zip file directly from the DoC website. Does such a disclaimer also exist on prose content from commerce.gov's web site?

**Comment by mike-charles** [comment]
Yeah I don't like that line, I think we still have a responsibility to protect those things.
